Import/Spreadsheet.ascx.cs

```diff
@@ -45,7 +45,7 @@ namespace RockWeb.Plugins.rocks_kfs.Import
     #region Block Settings
 
     [CustomDropdownListField( "Default Stored Procedure", "Select the default stored procedure to run when importing a spreadsheet.", "SELECT ROUTINE_NAME AS Text, ROUTINE_NAME AS Value FROM INFORMATION_SCHEMA.ROUTINES WHERE ROUTINE_TYPE = 'PROCEDURE' ORDER BY ROUTINE_NAME", false, "", "", 0 )]
-    [BooleanField( "Cleanup Table Parameter", "Select this if the SP has a @CleanupTable parameter.", true )]
+    [BooleanField( "Cleanup Table Parameter", "Select this if the SP has a @CleanupTable parameter and you want the procedure to cleanup the temp table from the import.", true )]
```

Comment:
Not that it matters that much, since this is just a manual block we deploy, but what if we don't have a CleanupTable parameter on our SP? It will fail having it have a default, won't it?

Reply:
Yes, that would be true...